cron-ci/setup-cron.sh

```diff
@@ -19,23 +19,27 @@ export HPC_STACK_FILE=config/stack_noaa.yaml
 
 # set machine name (hera, jet, orion, etc) so script edits correct ufs modulefiles
 export HPC_MACHINE_ID=
-export LOG_PATH=${HPC_HOMEDIR}/logs
+
+today=$(date +'%m-%d-%Y')
```

Comment:
can you make this`%Y-%m-%d`?
Will make sorting by year, month, day easier to look at.

Reply:
Ok